stake-pool/cli/src/main.rs

```diff
@@ -1648,8 +1948,8 @@ fn main() {
                     .help("Do not automatically merge transient stakes. Useful if the stake pool is in an expected state, but the balances still need to be updated."),
             )
         )
-        .subcommand(SubCommand::with_name("withdraw")
-            .about("Withdraw amount from the stake pool")
+        .subcommand(SubCommand::with_name("withdraw-stake")
+            .about("Withdraw amount from the stake pool into designated stake account")
```

Comment:
The destination stake account isn't required, so this addition could confuse people

Reply:
That's true. It would be nice to have the destination be an existing mergeable user stake account, however. 

The ephemeral stake account would be deleted once merged with the desination, in a single atomic transaction.

Reply:
Yeah that's been on my mental list for a bit... it would be a great feature